src/App/Controls/CircularProgressbarView.cs

```diff
@@ -21,13 +21,13 @@ public class CircularProgressbarView : SKCanvasView
             nameof(StrokeWidth), typeof(float), typeof(CircularProgressbarView), 3f);
 
         public static readonly BindableProperty ProgressColorProperty = BindableProperty.Create(
-            nameof(ProgressColor), typeof(Color), typeof(CircularProgressbarView), Color.Default);
+            nameof(ProgressColor), typeof(Color), typeof(CircularProgressbarView), Color.FromHex("175DDC"));
```

Comment:
Define the Color on the styles and then use it here to avoid hardcoding and maybe this color can be reused elsewhere

Reply:
This is only necessary on a case where the style is not defined (or missing the part that affects this). So putting it on a style could reproduce the error. 

In fact this color is from the default light style, the default color was resetting to white, not having it set was resetting to white. This was a kind of "improvement" if that case happens, using a color from the style, even if hardcoded

Reply:
But which is the error that it produces having color come from the Style here?

Reply:
It produces no error. This is a fail safe condition, this will only apply if not possible to get a color from styles.

Reply:
Alright, we'll leave it like this for now and I'll revise it later